recipe/post-link.bat

```diff
@@ -2,27 +2,57 @@
 setlocal ENABLEDELAYEDEXPANSION
 
 set menudir=%PREFIX%\Menu
+set menu=%menudir%\spyder-menu.json
+set logfile=%PREFIX%\.messages.txt
 
+rem  Check for conda-based install
 if exist "%menudir%\conda-based-app" (
-    :: Installed in installer environment, abridge shortcut name
+    rem  Abridge shortcut name
     call :patch " ^({{ ENV_NAME }}^)="
+
+    rem  Nothing more to do for conda-based install
+    goto :exit
 )
 
-if exist "%PREFIX%\condabin\" if exist "%PREFIX%\envs\" (
-    :: Installed in a base environment, use distribution name
-    call :patch "ENV_NAME=DISTRIBUTION_NAME"
+rem  Check for CONDA_PYTHON_EXE
+if not exist "%conda_python_exe%" (
+    rem  CONDA_PYTHON_EXE environment variable does not exist.
```

Comment:
Could you check if this env var is available when installing Spyder through Navigator? It should be, but better to be sure.

Reply:
Verified. I installed Anaconda3 as a separate distribution on my Windows VM. Using Anaconda Navigator, I installed Spyder 6.0.0a5.dev121 (my local build with this PR's changes). Testing both `menuinst` v1 and v2 in turn in the anaconda base environment, Spyder's shortcut was created correctly without errors.

Reply:
Nice! Thanks @mrclary.